### ISSUE TITLE
generalization failure

### ISSUE THREAD

**Issue by wzz-z** [bartn8/depthanyevent#6]
Dear author, thank you for your valuable contribution.
I have downloaded the pre-trained weights you released and conducted inference on samples from other datasets under the corresponding config files. To rule out the influence of resolution, I used the DAV_mvsec weight.
Notably, the DAV_mvsec config defaults to ViT-L, whereas the provided pre-trained weights are actually for ViT-S. I revised the config to ViT-S accordingly, but the achieved results are far from satisfactory. Could you please tell me what might cause this problem?

Below are my inference results.

<img width="660" height="471" alt="Image" src="https://github.com/user-attachments/assets/d7691fbc-d7b7-46bc-ac56-eca63e09e474" />

<img width="567" height="477" alt="Image" src="https://github.com/user-attachments/assets/d338df19-4095-4a14-aa72-ef6168c51703" />

**Comment by hangchenxie** [comment]
@wzz-z I also have similar problem. It seems the model have dense depth estimation, you need to use crop/mask/high contrast to show the objects depth, also need to try with different accumulated window. Overall, the best way I think is finetune the model to fit to our sparse scenarios.

**Comment by bartn8** [comment]
Hi, it seems like a very difficult case with low context. If you can acquire some RGB images along with events, you could apply the proxy distillation for your case.

Best,
Luca
